src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackImagePluginTest.java

```diff
@@ -68,12 +71,18 @@ public void testGetAllImages() throws FogbowManagerException, UnexpectedExceptio
     	
     	Map<String, String> expectedOutput = getPublicImages(generatedImages);
     	expectedOutput.putAll(getPrivateImagesFromProject(generatedImages, tenantId));
+    	
```

Comment:
hum ... It does not seem ok to have the getPrivateImagesFromProject duplicated in the code and here in the test.

Reply:
... talk to @fgan1 if you guys can refactor this fast, do so. Otherwise, I recommend adding a FIXME explaining the problema.

Reply:
Well, I solved this by creating a method that generates the expected images correctly, not requiring a post process.